docs/General/Manuals/client-manual.rst

```diff
@@ -1083,15 +1083,15 @@ Units View
 
 The :guilabel:`Units View` is a widget on the `Map View`_ that shows what units you have available and how
 many are being produced. The `Cities View`_ can tell you which cities are producing what unit. :numref:`Units
-View Button` shows the button to bring up the widget. Clicking the button a second time will close the widget.
+view button` shows the button to bring up the widget. Clicking the button a second time will close the widget.
 
-.. _Units View Button:
+.. _Units view button:
 .. figure:: ../../_static/images/gui-elements/top-bar-units.png
   :align: center
   :alt: Freeciv21 Units
   :figclass: align-center
 
-  Units View Button
+  Units view button
```

Comment:
Same as before, this is called "Units View" everywhere, I don't see why "View" is uncapitalized